fix: `--limit` options type (#1003)
lablup/backend.ai: changes/1003.fix.md

> Changed the value of `--limit` option to recognize as int type that was being treated as str.

Reviewer: We're changing "type" of `--limit` option, not "value". 😉
```suggestion
Fix `--limit` option being treated as `str` instead of `int`
```
